posthog/api/person.py

```diff
@@ -193,7 +193,7 @@ class PersonViewSet(StructuredViewSetMixin, viewsets.ModelViewSet):
     pagination_class = PersonCursorPagination
     filterset_class = PersonFilter
     permission_classes = [IsAuthenticated, ProjectMembershipNecessaryPermissions, TeamMemberAccessPermission]
-    lifecycle_class = ClickhouseLifecycle
+    lifecycle_class = Lifecycle
     retention_class = ClickhouseRetention
```

Comment:
Hmm, looking at so many changes now, and seeing how trends doesn't have an enterprise variant, I kinda feel we botched naming here a bit, since `ClickhouseRetention` and `Retention` both are infact clickhouse based.

Maybe we should've named them `EnterpriseRetention` and `Retention` instead, like we did with `EventQuery`, and `EnterpriseEventQuery` ?